rocAL/rocAL/source/decoders/image/fused_crop_decoder.cpp

```diff
@@ -30,24 +30,27 @@ FusedCropTJDecoder::FusedCropTJDecoder(){
 
 #if 0
```

Comment:
This case will never get invoked #if 0
Is there any MACRO needs to be used instead of 0?

Reply:
This was already part of the file
I guess I need to check with Shobana on this.